.github/workflows/backport.yml

```diff
@@ -39,5 +39,3 @@ jobs:
           github_token: ${{secrets.KIBANAMACHINE_TOKEN}}
           commit_user: kibanamachine
           commit_email: 42973632+kibanamachine@users.noreply.github.com
-          auto_merge: 'true'
-          auto_merge_method: 'squash'
```

Comment:
These values are already set in https://github.com/elastic/kibana/blob/main/.backportrc.json